packages/babel-preset-cgb/index.js

```diff
@@ -30,35 +29,38 @@ module.exports = {
 		],
 	],
 	plugins: [
-		// class { handleClick = () => { } }.
-		[ require.resolve( 'babel-plugin-transform-class-properties' ) ],
-		// The following two plugins use Object.assign directly, instead of Babel's
-		// extends helper. Note that this assumes `Object.assign` is available.
-		// { ...todo, completed: true }
+		'@babel/plugin-syntax-dynamic-import',
+		'@babel/plugin-syntax-import-meta',
 		[
-			require.resolve( 'babel-plugin-transform-object-rest-spread' ),
+			'@babel/plugin-proposal-object-rest-spread',
 			{
 				useBuiltIns: true,
 			},
 		],
 		[
-			// Transforms JSX Syntax.
-			// 'transform-react-jsx',
-			require.resolve( 'babel-plugin-transform-react-jsx' ),
+			'@babel/plugin-proposal-class-properties',
 			{
-				pragma: 'wp.element.createElement',
```

Comment:
Why removing the pragma? It won't work without this.

Reply:
I've now added a plugin that handles this(@wordpress/babel-plugin-import-jsx-pragma), instead of the pragma option on a plugin we are no longer including (the @babel/react handles this).

Reply:
I don't understand this. Do we not need it anymore? Why? Since WP does use a pragma option here. I am happy to drop it if there's no need.

Reply:
@ahmadawais Digging even deeper I found this:

https://github.com/WordPress/gutenberg/blob/master/packages/babel-preset-default/index.js

Which appears to include what we need here. It includes the pragma settings. I'm beginning to test this now. It's weird that in my testing I was able to create blocks and insert them into posts without this part of the config. I'm going to spend a bit more time testing around this.